plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -424,7 +424,6 @@ func (c *CloudWatch) BuildMetricDatum(point telegraf.Metric) []*cloudwatch.Metri
 	//https://www.ardanlabs.com/blog/2013/08/understanding-slices-in-go-programming.html
 	var datums []*cloudwatch.MetricDatum
 	for k, v := range point.Fields() {
-		log.Printf("D! #Fields of the current point: k - %s v - %f", k, v)
```

Comment:
Why was the log removed?

Reply:
I added this log in previous PR for debugging purpose(more insights about what fields each data point contains), but end up finding it will become really noisy when rolling up dimensions in debug mode. Because the log happen every time the rolling happens.